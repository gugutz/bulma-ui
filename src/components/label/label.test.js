import React from 'react'
import renderer from 'react-test-renderer'
import Label from './label'

describe('<Label /> component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Label>cool label</Label>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
