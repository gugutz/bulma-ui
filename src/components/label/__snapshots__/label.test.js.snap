// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Label /> component should render correctly 1`] = `
<label
  className="label"
>
  cool label
</label>
`;
